specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+
+Given that, we should consider options for card author label formatting. Consider the two following option.
+
+### Allow TextBlocks and RichTextBlocks as Labels
+
+One option would be to add support for TextBlock and/or RichTextBlock labels to add formatting support:
+
+```json
+{
+	"type": "Input.Text",
+	"id": "nameInput",
+	"label": 
+	{
+		"type": "TextBlock",
+		"text": "Please enter your name:",
+		"weight": "Bolder",
+		"color": "Good"
+	},
+	"placeholder": "Last, First"
+}
+```
+
+The problem with this approach, however, is that if we allow the host to specify formatting, there could be a conflict between the formatting provided by the card author and that provided by the host. For example, if the card author makes all their labels bold, the host's indication that required labels should be bold will not provide any additional information, and the end user will not see any indication of which fields are required.
+
+##### Non-adjacent labels
+
+Following this approach, non-adjacent labels are not going to be possible as labels are going to be tied to the inputs which will make it easier to users to know what label corresponds to each input.
+
+As this approach sets the rules of how labels are rendered, it also limits where a label can be drawn in valid scenarios as the next ones:
+
+![img](assets/InputLabels/LabelsToTheLeft.png)
+
+I built this card using a series of column sets. Although it is an acceptable UI, this is probably not the approach we want for card authors to label their inputs on the side. The card authoring is fairly non-intuitive to get a good result. If we want to support labels to the left of the input, we should probably do this as a first-class explicit property (```"labelPosition"``` or ```"allowHorizontalLabels"```). 
+
+This card is another possible example of a use case for non-adjacent labels, where there is additional information between the label and the input:
+
+![img](assets/InputLabels/AdditionalLabelContent.png)
+
+This scenario could be covered by a RichTextBlock that contains both pieces of text being used as the label (arguably the entire text should be read as the accessibility label for the input). 
+
+This approach will limit what elements can be rendered as the label to only text.
+
+## Backwards Compatibility
+On earlier versions of renderers the label would be completely dropped, leading to a bad user experience. The card may be completely unusable if without the labels the user cannot deduce what information is meant to be provided in the unlabeled inputs.
```

Comment:
Hmm this one indeed sounds precarious and from what @paulcam206 demo'd to me  quite confusing indeed. On the other hand this option indeed seems to be the most intuitive :( -- Isnt there some novel use of fallback on hte new cards that could be leveraged? @paulcam206 ?

Reply:
It's going to impact new cards that use the label property on inputs rendering on say older versions of Teams hosts etc -- but i wonder if the newer cards can include feature fallback that shipped in 1.2 and we provide that guidance?

Reply:
Aah never mind - i see your note around fallback down below! thanks